docs/upgrading.md

```diff
@@ -7,6 +7,12 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open AEA
 
+## `v1.16.0` to `v1.17.0`
+
+No backwards incompatible changes.
+
+Plugins from previous versions are not compatible anymore.
+
```

Comment:
Don't we need to remove some file to upgrade from 1.15 to 1.16? Not sure, but yesterday I think david m mentioned something during the Brahma call, although I'm not sure if it was for open-aea or open-autonomy.

Reply:
There were no major changes on open-aea except for dockerfiles, so I'm think he might be talking about removing the old CLI config since we have added the schema validation for it.

Reply:
Yes, below (## `v1.15.0` to `v1.16.0`) needs updating similarly as I did here: https://github.com/valory-xyz/open-autonomy/pull/1218/commits/1e16ba6432987188880bb8e0fef05eeb42e0902e